Updated Gemfile and Environment Variables (#4)
Bandwidth-Samples/voice-bridge-ruby: README.md

# Environmental Variables
The sample app uses the below environmental variables.
```sh
BW_ACCOUNT_ID               	     # Your Bandwidth Account Id

> BW_USERNAME                  	     # Your Bandwidth API Token

Reviewer: ```suggestion
BW_USERNAME                  	     # Your Bandwidth API Username
```

BW_PASSWORD                     # Your Bandwidth API Password
BW_NUMBER                       # Your Bandwidth Phone Number

> USER_NUMBER                     # The user's phone number involved with this application

Reviewer 2: ```suggestion
USER_NUMBER                     # The phone number to contact, provided by the user
```
I thought the wording was a bit iffy... which do you think makes more sense, @hamermike @ajrice6713 ?
